webextensions/api/tabs.json

```
@@ -4175,7 +4168,7 @@
               "description": "`updateProperties.url` parameter",
               "support": {
                 "chrome": {
-                  "version_added": "≤61"
+                  "version_added": "18"
```

:x: Same as `update.updateProperties_pinned_parameter`:

```suggestion
                  "version_added": "9"
```